### ISSUE TITLE
Add support for Django 2.2

### ISSUE THREAD

**Issue by yisraell** [3YOURMIND/django-add-default-value#14]
Is there a reason this is limited to Django<2.1

Also, the related `django-migration-linter` tests for `django_add_default_value`, and it doesn't have that same limitation...

**Comment by flixx** [comment]
Out of my sight, the only reason why we didn't add support for Django 2.2 is, that nobody tested it with Django 2.2 so far. Right @David-Wobrock @mes3yd ?

@yisraell I think it would help a lot, if you could test it with Django 2.2 and confirm that it is working.

**Comment by yisraell** [comment]
I haven't actually had a need to use it in my code yet :)
I just started using `django-migration-linter` recently and I understood that this package is often useful with it - I just wanted to be able to use it later when I need it...
Maybe  there is someone else who is currently using it that could test it with 2.2?

**Comment by flixx** [comment]
@yisraell okay. Then let's wait a bit until s.o. can test.
We will probably soon (1-2 months?) switch to Django 2.2, then I could do the testing.

**Comment by POD666** [comment]
any update?

**Comment by Ylodi** [comment]
I tested code with Django 2.2 and Python 3.7, 3.8 (PostgreSQL database), tests passing. I can also test MySQL, but I can't test with MSSQL.

**Comment by igeligel** [comment]
Is this a platform issue? If so I would recommend using https://hub.docker.com/_/microsoft-mssql-server which should run on Unix systems as well.

Maybe even create a guide in the README to showcase to test the different database providers.

**Comment by flixx** [comment]
Hello @yisraell @POD666 @Ylodi @igeligel 

The projects maintainer switched (to me)...
I added Django 2.2 and 3.0 support.

However, I was also not able to test things with MSSQL, so I decided, to mark it as untested in the README (and remove the MSSQL environments in tox) until we find somebody who can support it further.

Supporting the new Django versions seems to be more important than to keep the package working with MSSQL as well.
